spirv_glsl.cpp

```diff
@@ -2890,8 +2939,7 @@ void CompilerGLSL::declare_undefined_values()
 		if (options.force_zero_initialized_variables && type_can_zero_initialize(type))
 			initializer = join(" = ", to_zero_initialized_expression(undef.basetype));
 
-		statement(variable_decl(type, to_name(undef.self), undef.self), initializer,
-		          ";");
+		statement(variable_decl(type, to_name(undef.self), undef.self), initializer, ";");
```

Comment:
Fallout from using formatting script?

Reply:
@HansKristian-Work want anything done about it?

Reply:
Ideally, unrelated formatting should not be part of this PR. I'll probably need to do a "format_all.sh" commit soon.